src/lib/enums/general.enums.ts

```diff
@@ -3,3 +3,11 @@ export enum Action {
     Edit = "edit",
     Add = "add",
 }
+
+export enum LocalStorageKey {
+    Chains = "chains",
+    PersistedDataVersion = "persisted-data-version",
+    Theme = "theme",
+    ShowHexAsText = "show-hex-as-text",
```

Comment:
this key has a new name, we should apply a migration to it 🙏🏼 currently its getting duplicated, users might get the old data + the new data